db/migrate/20210609041826_add_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNameToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :firstname, :string
```

Comment:
If its mandatary field, can we had `null: false`

Reply:
I believe it is not mandatory (at least for the sake of Keywords searching). 
But good to know ;-)